### PR TITLE
More initialize() fixes

### PR DESCRIPTION
**Pull request by DamianX**
thanks oldcoders

### CODE REVIEW

File: code/game/machinery/computer/arcade/arcade.dm [hunk 1]
```diff
@@ -5,14 +5,20 @@
...
 	emag_cost = 0 // because fun
 	light_color = LIGHT_COLOR_GREEN
 
 
-/obj/machinery/computer/arcade/New()
-	..()
-	game = new /datum/arcade_game/space_villain(src)
+/obj/machinery/computer/arcade/initialize()
+	. = ..()
+	update_game()
```
Comment (MadmanMartian) [inline code/game/machinery/computer/arcade/arcade.dm:16]: thanks for catching this one for me

File: code/game/machinery/computer/arcade/arcade_game.dm [hunk 2]
```diff
@@ -193,7 +193,7 @@
 				new /obj/item/device/maracas/cubanpete(holder.loc)
 				message_admins("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
 				log_game("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
-				holder.New()
+				holder.update_game()
```
Comment (MadmanMartian) [inline code/game/machinery/computer/arcade/arcade_game.dm:196]: Dont necessarily want it to qdel the old game, just reset it.

Could want it to hold high scores

Comment (DamianX) [reply code/game/machinery/computer/arcade/arcade_game.dm:196]: I assumed that was the intent of the old version; am I reading that wrong or was the old version just overwriting the datum anyway?
I'll change it but I want to make sure

Comment (MadmanMartian) [reply code/game/machinery/computer/arcade/arcade_game.dm:196]: Oh damn, that does seem to be the intent. That sucks. Disregard my comment here.

**Pull request merged**